.gitignore

```diff
@@ -309,3 +309,4 @@ static
 
 # Next.js
 .next
+/src/frontend/playwright/storage/
```

Comment:
you already have playright storage in the frontend gitignore file, do you need it here too?

Reply:
mmm actually these changes were already in, i think i might changed 2 files only